src/components/application_manager/src/resumption/resumption_data_processor.cc

```diff
@@ -473,6 +489,24 @@ void ResumptionDataProcessor::AddFiles(
   }
 }
 
+void ResumptionDataProcessor::AddWindows(
+    application_manager::ApplicationSharedPtr application,
+    const ns_smart_device_link::ns_smart_objects::SmartObject& saved_app) {
+  using namespace mobile_apis;
```

Comment:
Won't see usage any form mobile_api

Reply:
@LuxoftAKutsan good point. Me too. Removed in https://github.com/LuxoftSDL/sdl_core/pull/70/commits/6b8a138a35e6abea6bf9b6841a0d21d96c2bc431